dask_searchcv/model_selection.py

```diff
@@ -843,11 +898,20 @@ def visualize(self, filename='mydask', format=None, **kwargs):
 
 {parameters}
 
-scoring : string, callable or None, default=None
-    A string (see model evaluation documentation) or
-    a scorer callable object / function with signature
-    ``scorer(estimator, X, y)``.
-    If ``None``, the ``score`` method of the estimator is used.
+scoring : string, callable, list/tuple, dict or None, default: None
+    A single string (see :ref:`scoring_parameter`) or a callable
+    (see :ref:`scoring`) to evaluate the predictions on the test set.
+
+    For evaluating multiple metrics, either give a list of (unique) strings
+    or a dict with names as keys and callables as values.
+
+    NOTE that when using custom scorers, each scorer should return a single
+    value. Metric functions returning a list/array of values can be wrapped
+    into multiple scorers that return one value each.
+
+    See :ref:`multimetric_grid_search` for an example.
```

Comment:
These refs don't exist in our docs, please remove them.

Reply:
Happy to remove them, but do you want me to setup intersphinx so that they do exist?

Reply:
Up to you, I've never used intersphinx before. If you think it aids in clarity, then sure - perhaps tweak the surrounding language to indicate this is a scikit-learn example/reference instead of a `dask-searchcv` example?

Reply:
Just removed for now.